pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -458,8 +461,12 @@ protected void internalDeleteNamespaceForcefully(AsyncResponse asyncResponse, bo
             for (NamespaceBundle bundle : bundles.getBundles()) {
                 // check if the bundle is owned by any broker, if not then we do not need to delete the bundle
                 if (pulsar().getNamespaceService().getOwner(bundle).isPresent()) {
-                    futures.add(pulsar().getAdminClient().namespaces()
-                            .deleteNamespaceBundleAsync(namespaceName.toString(), bundle.getBundleRange(), true));
+                    CompletableFuture<Void> futureForDeleteBundle = FutureUtil.waitForAll(futuresForDeleteTopics)
```

Comment:
Forcibly delete the order of topics and bundles, becase after deleted bundles, the topic will be fenced, then the operation that delete topic will failure